x-pack/plugins/security_solution/public/detections/components/alerts_table/translations.ts

```diff
@@ -136,6 +136,13 @@ export const ACTION_ADD_ENDPOINT_EXCEPTION = i18n.translate(
   }
 );
 
+export const ACTION_ADD_RANSOMWARE_EXCEPTION = i18n.translate(
+  'xpack.securitySolution.detectionEngine.alerts.actions.addRansomwareException',
```

Comment:
I don't think this is needed. We could continue to use the `Add Endpoint Exception` title for these modals